### ISSUE TITLE
compute button no longer seems to work (Trac #753)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#869]
after closing #823 it seems that something along the way has broken that button completely - basically it no longer seems to do anything regardless of being theory or a fit page with data.  Does seem that in 2D it may be working.

Migrated from http://trac.sasview.org/ticket/753
```json
{
    "status": "closed",
    "changetime": "2016-12-19T18:52:37",
    "_ts": "2016-12-19 18:52:37.874089+00:00",
    "description": "after closing #694 it seems that something along the way has broken that button completely - basically it no longer seems to do anything regardless of being theory or a fit page with data.  Does seem that in 2D it may be working.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-10-10T14:10:35",
    "component": "SasView",
    "summary": "compute button no longer seems to work",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "ajj",
    "type": "defect"
}
```

**Comment by ajj** [comment]
Trac update at `2016/10/10 20:01:16`: 
* **ajj** commented:

In changeset 8662a587350e2b5ad37342ca1d19a5aeb0e7fe8c:

```text
#!CommitTicketReference repository="sasview" revision="8662a587350e2b5ad37342ca1d19a5aeb0e7fe8c"
Updating basepage and fitpage to pass is_modified flag out of helper functions so that _onDraw can correctly determine that there is a change and recalculate/redraw

Fixes #869 compute button no longer seems to work
```


* **ajj** changed resolution from "" to "fixed"
* **ajj** changed status from "new" to "closed"

**Comment by ajj** [comment]
Trac update at `2016/10/10 20:48:06`: **ajj** commented:

In changeset 3bec50ae2f0e34d160b7c5524ce22736baa10388:

```text
#!CommitTicketReference repository="sasview" revision="3bec50ae2f0e34d160b7c5524ce22736baa10388"
Really fixes #869 (proper call to !wx.CallAfter that works outside if a debugger)
```

**Comment by butlerpd** [comment]
Trac update at `2016/10/10 20:48:06`: **butler** changed _comment0 from:

> In [changeset:"3bec50ae2f0e34d160b7c5524ce22736baa10388/sasview"]:
> {{{
> #!CommitTicketReference repository="sasview" revision="3bec50ae2f0e34d160b7c5524ce22736baa10388"
> Really fixes #753 (proper call to wx.CallAfter that works outside if a debugger)
> }}}

to:

> 1477583411024622

**Comment by pkienzle** [comment]
Trac update at `2016/10/13 05:11:48`: 
* **pkienzle** commented:

Select a model, change Npts to 500 and click "Compute".  On the mac nothing happens.  Press "Enter" in the Npts field and the theory is updated.

Toggling "Log?" does not trigger a recompute (good), but the after toggling "Log?" the compute button doesn't change anything.  

Compute does recognize a change in a parameter value and update accordingly.


* **pkienzle** changed resolution from "fixed" to ""
* **pkienzle** changed status from "closed" to "reopened"
